.github/workflows/required-checks-workflow.yml

```
@@ -2,14 +2,16 @@ name: Required checks
 
 # Run this check any time a PR is opened, synchronized, reopened,
 # labeled or unlabeled.
+# Additionally, run this check when a PR reviewer submits their review.
 on:
   pull_request:
     types: [opened, synchronize, reopened, labeled, unlabeled]
+  pull_request_review:
+    types: [submitted]
 
 jobs:
   do_not_merge:
     name: Do not merge
     runs-on: ubuntu-latest
     steps:
-      - run: echo "::error ::PR can't be merged with the DO NOT MERGE label."; exit 1
-        if: contains(github.event.pull_request.labels.*.name, 'DO NOT MERGE')
\ No newline at end of file
+      - run: ${{ contains(github.event.pull_request.labels.*.name, 'DO NOT MERGE') && echo '::set-failed' }}
```

```suggestion
      - run:  echo "::set-failed" 
        if: contains(github.event.pull_request.labels.*.name, 'DO NOT MERGE') }}
```
👋 Looks like you have a workflow error, https://github.com/GoogleChrome/web.dev/actions/runs/108564511

When there's an error parsing the YAML, the status check doesn't always make it back to the PR